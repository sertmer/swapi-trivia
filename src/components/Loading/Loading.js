import React from 'react'
import './Loading.scss'

const Loading = () => {
  return (
    <div>
       <img
          className='loading'
          src={'https://media.giphy.com/media/W5s26R38g69Uc9CWCI/giphy.gif'}
          alt={''}
        />
      <p>Hang tight...delivering your data</p> 
    </div>
  )
}

export default Loading