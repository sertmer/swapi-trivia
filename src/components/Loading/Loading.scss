.loading {
  width: 100vh;
  height: auto;
}