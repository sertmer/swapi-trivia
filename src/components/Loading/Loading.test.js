import React from 'react'
import { shallow } from 'enzyme'
import Loading from './Loading'

describe('Loading', () => {
  let wrapper
  
  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})