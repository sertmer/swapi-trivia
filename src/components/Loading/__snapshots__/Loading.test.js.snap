// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
<div>
  <img
    alt=""
    className="loading"
    src="https://media.giphy.com/media/W5s26R38g69Uc9CWCI/giphy.gif"
  />
  <p>
    Hang tight...delivering your data
  </p>
</div>
`;
